packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -380,6 +380,9 @@ Future<dynamic/*=T*/> showMenu/*<T>*/({
 /// its menu to be dismissed.
 typedef void PopupMenuItemSelected<T>(T value);
 
+/// Signature used by [PopupMenuButton] to lazily construct the items shown when the button is pressed.
+typedef List<PopupMenuEntry/*<T>*/> PopupMenuItemBuilder/*<T>*/(BuildContext context);
```

Comment:
Actually what am I saying. typedefs already support generics. This can just be:
```dart
typedef List<PopupMenuEntry<T>> PopupMenuItemBuilder<T>(BuildContext context);
```